lib/native/mqtt.ts

```diff
@@ -655,8 +665,6 @@ export class MqttClientConnection extends NativeResourceMixin(BufferedEventEmitt
         if (error_code == 0 && return_code == 0) {
             resolve(session_present);
             this.emit('connect', session_present);
-            let successCallbackData = { session_present: session_present, reason_code: return_code } as OnConnectionSuccessResult;
-            this.emit('connection_success', successCallbackData);
         } else if (error_code != 0) {
             reject("Failed to connect: " + io.error_code_to_string(error_code));
             let failureCallbackData = { error: new CrtError(error_code) } as OnConnectionFailedResult;
```

Comment:
We probably wanna remove `connection_failure` as well, as now it has its own callback.